doc/manpages.py

```diff
@@ -165,6 +165,11 @@
     ('man3/flux_log', 'flux_log_set_procid', 'Log messages to the Flux Message Broker', [author], 3),
     ('man3/flux_log', 'flux_log', 'Log messages to the Flux Message Broker', [author], 3),
     ('man3/flux_msg_cmp', 'flux_msg_cmp', 'match a message', [author], 3),
+    ('man3/flux_msg_create', 'flux_msg_create', 'create a Flux message', [author], 3),
+    ('man3/flux_msg_create', 'flux_msg_copy', 'copy a Flux message\'s attributes into a new Flux message', [author], 3),
+    ('man3/flux_msg_create', 'flux_msg_incref', 'get a memory address for a Flux message and increment reference count', [author], 3),
+    ('man3/flux_msg_create', 'flux_msg_decref', 'decrement the reference count of a Flux message by 1', [author], 3),
+    ('man3/flux_msg_create', 'flux_msg_destroy', 'destroy a Flux message when reference count reaches 0', [author], 3),
```

Comment:
I'd go shorter on the descriptions.  If you want to be lazy, use the same description for all the stubs since that's the  convention.   At least try to avoid a line break in the ascii rendering if at all possible.  Check with e.g.

```
cd doc/man3
make -C ..
man ./flux_msg_destroy.3
```

For example just use "destroy a Flux message"  or "take a reference on a Flux message".